data/json/monsters/monsters.json

```diff
@@ -532,6 +533,43 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_fused",
+    "type": "MONSTER",
+    "name": "fused",
```

Comment:
I´d suggest something like "fused horror", "fused one" or even "skinwalker". Just calling it fused seems strange to me.

Reply:
Hi. The name was since we have a 'boomer' and i kinda thought short 'descriptive' zombie names have their charm.

Reply:
thought a bit: how about "dissoluted devourer"?